### ISSUE TITLE
Longer term issue: Integration with Hackey Trackey and Hackey Patterns

### ISSUE THREAD

**Issue by JoepVanlier** [JoepVanlier/Hackey-Machines#7]
Ultimately, for a self-contained vibe in this environment, a way to call Hackey Trackey (and alternatively the standard piano roll editor) for the currently selected track / device, select/insert/rearrange MIDI items on that track, and edit them right then and there... Would wrap this up as a full-fledged Buzz frontend for Reaper . I know, I know, it's kind of a tall order, hahh. And mostly fueled by pure nostalgia.

**Comment by michaelsjackson** [comment]
Yes, but we need Buzz in Reaper. Thanks sai'ke! This is a dream combination.
Playing frontend: Sequetron going to BlueARP in Reaper.
Editing frontend: Buzz in Reaper.
